WrightTools/data/_data.py

```diff
@@ -742,7 +742,7 @@ def map_variable(self, variable, points, input_units='same', *, name=None, paren
         if self.ndim == 1:
 
             def interpolate(dataset, points):
-                function = scipy.interpolate.interp1d(variable[:], dataset[:])
+                function = scipy.interpolate.interp1d(variable[:], dataset[:], bounds_error=False)
```

Comment:
Unrelated?

Reply:
yes will you let me sneak it in :kissing_heart: